docs/conf.py

```diff
@@ -5,7 +5,11 @@
 author = lablib.__author__
 release = lablib.__version__
 
-extensions = ["sphinx.ext.autodoc"]
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "revitron_sphinx_theme"
```

Comment:
i'm struggling a bit with getting `revitron` to build.
always getting something related to this: https://github.com/readthedocs/sphinx_rtd_theme/issues/1465